changelog/next/features/4270--mpmc-pub-sub.md

```diff
@@ -0,0 +1,13 @@
+The `publish` operator's topic must no longer be unique. Instead, any number of
```

Comment:
```suggestion
The `publish` operator's topic no longer have to be unique. Instead, any number of
```

Reply:
With "must not" you basically say they can't be unique. Not what you want, I think.

Reply:
Good point, not sure what I was thinking yesterday while writing this. 😅